src/librustc/ty/context.rs

```diff
@@ -1024,7 +1026,11 @@ pub struct GlobalCtxt<'tcx> {
     /// Export map produced by name resolution.
     export_map: FxHashMap<DefId, Lrc<Vec<Export>>>,
 
-    hir_map: hir_map::Map<'tcx>,
+    pub hir_forest: hir::map::Forest,
```

Comment:
The "forest" name doesn't make sense anymore, since cross-crate HIR inlining was removed, I think we should call this `DepTrackedCrate` or just keep the `hir::Crate` in here.